Backpack-SwiftUI/Dialog/Classes/BPKDialog.swift

```diff
@@ -27,6 +27,7 @@ public extension View {
         text: String,
         confirmButton: BPKDialogButton,
         secondaryActions: BPKDialogSecondaryActions? = nil,
+        rootViewController: UIViewController? = nil,
```

Comment:
I wonder if these need to be required (non optional), as if you call this BPKDialog with a nil rootViewController it will just not be shown. If this is the case, the API would be allowing broken behaviour to be defined.

When this was internal code, then it made more sense for the method that returned it to be optional, but now that this is exposed we should allow for no invalid state to be sent (ie, a nil rootViewController)

Reply:
yeah I think we need to make it a breaking change here and fix the parts in the app that use this (only 1 I believe in Hotels?)

Reply:
Updated: 
be2494a72253b7e7b25cea483a76febc6a2f2dae